news/2025/2025-10-30-fall-time-mania-mapping-contest.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Fall Time Mania Mapping Contest 2025
+date: 2025-10-30 16:00:00 +0000
+---
+
+The first iteration of FTMC is on it's way with the chilly autumn air! Are you ready to start your holiday mapping?
+
+![](https://i.gyazo.com/9356eb561bd54282db105a324669c887.png)
+
+The **Fall Time Mania Mapping Contest** is debuting this year and with the temperature already dropping, we would like to give you a warm welcome in this exhilarating contest! You can read the full details in the forum post linked [here]()!
+
+This mapping contests stems from it's close seasonal partner, [STMI](https://osu.ppy.sh/community/forums/topics/2050785?n=1), which targeted a specific mapping demographic being an only 6K keymode mapping contest without gimmick to try and get more people interested in the keymode which was a complete success, but this time around, all mappers are welcomed and we hope to see your entry soon!
+
+FTMC will be a one difficulty, one song per submission contest from osu!'s Featured Artist listing. It will follow a specific judging criteria listed below and also expanded on in the [contest's forum post found here](https://osu.ppy.sh/community/forums/topics/2145873?n=1). How should you map? Well, the editor is your canvas! You're allowed to map any style as long as it adheres to the rules and to the gimmick!
```

Comment:
Couple things:

- The previous (original) paragraph claimed the contest has no gimmicks. Having an explicit mention of "gimmick" in the criteria, and another one here, seems to be somewhat contradictory to it. Would stick to one or the other, but it probably would be best not to claim there are no gimmicks if the word stays in the current criteria.
- This feels like a rather compact, hard to read summary of the entire ruleset. Would've probably opted by focusing on the important stuff. Right now, this paragraph feels like it's trying to cramp *too* much information at the same time. Though this could most likely be a nitpick.